parse/wasm/wasm.go

```diff
@@ -1,18 +1,18 @@
 //go:build js && wasm
 
-//go:generate env GOOS=js GOARCH=wasm go build -o kuneiform.wasm -ldflags "-s -w" -trimpath -tags netgowasm.go
+//go:generate env GOOS=js GOARCH=wasm go build -o kuneiform.wasm -ldflags "-s -w" -trimpath -tags netgo wasm.go CGO_ENABLED=0
```

Comment:
`CGO_ENABLED=0` would go with the other `env` settings like `GOOS` rather.

Reply:
whoops thanks. Sorry, been a long day